addons/safeStart/functions/fnc_missionTimeWarning.sqf

```diff
@@ -19,15 +19,26 @@ if !(isServer) exitWith {};
 
 params [];
 
+private _missionLength = getMissionConfigValue [QEGVAR(missionTesting,missionTimeLength),0];
+private _missionLengthSec = _missionLength * 60;
```

Comment:
on old missions it's getting a string
`Error *: Type String, expected Number,Not a Number`